src/main/kotlin/com/dp/advancedgunnerycontrol/typesandvalues/WeaponAITags.kt

```diff
@@ -43,18 +43,21 @@ val tagTooltips = mapOf(
     "AvdShields+" to "As AvoidShields, but will never fire when shields are up and not flanked. (experimental)",
     "NoFighters" to "Weapon won't target fighters.",
     "ConserveAmmo" to "Weapon will be much more hesitant to fire when ammo below ${(Settings.conserveAmmo()*100f).toInt()}%.",
+    "ConservePDAmmo" to "Weapon will only fire at maximum ROF when the target is of type PD and ammo is below ${(Settings.conserveAmmo()*100f).toInt()}%.",
     "Opportunist" to "Weapon will be more hesitant to fire and won't target missiles or fighters. Use for e.g. limited ammo weapons.",
     "AvoidDebris" to "Weapon will not fire when the shot is blocked by debris/asteroids." +
             "\nNote: This only affects the custom AI and the Opportunist mode already includes this option.",
     "BigShips" to "Weapon will ignore missiles and prioritize big ships" +
             if(Settings.strictBigSmall()) " and won't fire at anything smaller than destroyers." else "",
-    "SmallShips" to "Weapon will ignore missiles and prioritize small ships" +
+    "SmallShips" to "Weapon will ignore missiles and prioritize small ships (including fighters)" +
```

Comment:
just added this as i personally was unsure of this fact until i looked at the code